client/src/components/Modals/Modal.js

```diff
@@ -31,18 +32,30 @@ const BadgeContainer = styled(RemoveContainer)`
 `;
 
 export default function DictonaryList({ onClick }) {
-  return (
-    <>
-      <ModalContainer>
-        <RemoveContainer>
-          <button onClick={() => onClick()}>
-            <RemoveIcon />
-          </button>
-        </RemoveContainer>
-        <BadgeContainer>
-          <Badge>Hallo</Badge>
-        </BadgeContainer>
-      </ModalContainer>
-    </>
-  );
+  let words = [];
+  words = useFetch('http://localhost:7100/api/auth');
```

Comment:
The line above is not required, because `words` get a new value from `useFetch`.

```suggestion
  const words = useFetch('http://localhost:7100/api/auth');
```

And you should add a proxy (not in this pull request).

Reply:
The naming is confusing.
Why do you receive `words` from your `auth` api? 
I would expect to receive a user or auth object. 

And `words` sounds like an array, but it is a object.